apiserver/pkg/manager/resource_manager.go

```diff
@@ -104,12 +104,12 @@ func (r *ResourceManager) populateComputeTemplate(ctx context.Context, cluster *
 	return dict, nil
 }
 
-func (r *ResourceManager) GetCluster(ctx context.Context, clusterName string) (*v1alpha1.RayCluster, error) {
+func (r *ResourceManager) GetCluster(ctx context.Context, clusterName string, namespace string) (*v1alpha1.RayCluster, error) {
 	if len(clusterName) == 0 {
 		return nil, util.NewInvalidInputError("clusterName is empty, failed to get the cluster.")
 	}
 
-	client := r.getRayClusterClient(DefaultNamespace)
+	client := r.getRayClusterClient(namespace)
```

Comment:
1. Do we want to support default namespace? 
2. Is namespace a required field?

Reply:
I was thinking whether we like to support backward compatibility? In our interval version, `namespace` is still optional, if user doesn't specify it, we store the CR in the default namespace. If they do, we then use the namespace. In upstream version, this is still under active development, so I think either way works.

Reply:
>Is namespace a required field?

Namespace is a required field in protos, but api server set the default namespace when it's empty for `create` op. I'll follow the existing logic for `get` and `delete` ops as well.

>Do we want to support default namespace?

I think either way would work for us. Maybe we can make this default namespace configurable in the future.

Reply:
Set `namespace` to `DefaultNamespace` [here](https://github.com/ray-project/kuberay/pull/229/files#diff-6d24c4f57fa68eaf6e5d018e13cb8549476d9047c9bd3cf8f496a5760e8bc3b9R112)